File: engine/model/src/main/scala/tools/packet/character/box/UseChalkboard.scala
package tools.packet.character.box

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class UseChalkboard(private var _characterId: Int, private var _close: Boolean, private var _text: String) extends PacketInput {
  def characterId: Int = _characterId

  def close: Boolean = _close

  def text: String = _text

  override def opcode(): SendOpcode = SendOpcode.CHALKBOARD
}

File: engine/engine-model/src/main/java/tools/packet/character/box/UseChalkboard.java
package tools.packet.character.box;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record UseChalkboard(Integer characterId, Boolean close, String text) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.CHALKBOARD;
   }
}